modules/gitops-release-notes-for-1-10-3.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-title-1-10-3_{context}"]
+= Release notes for {gitops-title} 1.10.3
+
+{gitops-title} 1.10.3 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-10-3_{context}"]
+== Errata updates
+
+[id="rhsa-2024-128924-02-gitops-1-10-3-security-update-advisory_{context}"]
+=== RHSA-xxxx:nnnn - {gitops-title} 1.10.3 security update advisory
```

Comment:
@Dhruv-Soni11 FYI and FYA:

NOTICE: [RHSA-2024:128924-02](https://errata.engineering.redhat.com/advisory/128924) has changed its advisory name to [RHSA-2024:1345-02](https://errata.engineering.redhat.com/advisory/128924)

Please update the PR and ping me, I will check and merge the RN and publish it. Thank you!

Reply:
Fixed it now @Srivaralakshmi. PTAL.